testing/trino-product-tests/src/main/java/io/trino/tests/cli/TestTrinoCli.java

```diff
@@ -374,6 +417,40 @@ private void launchTrinoCliWithRedirectedStdin(File inputFile)
         trino = new TrinoCliProcess(processBuilder.start());
     }
 
+    private Map<String, String> getTrinoCliArguments(Map<String, String> arguments)
```

Comment:
Do we need this one? Can't we transform list generated by `private List<String> getTrinoCliArguments(String... arguments)` to map?

Reply:
we could by partitioning list but I don't like that approach